modules/getting-started-cli-creating-route-backend.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * tutorials/dev-app-cli.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="getting-started-cli-creating-route-backend_{context}"]
+= Exposing the back-end service
+
+Similar to how you exposed the front-end service for external clients, you must now expose the back-end service by creating a route.
+
+.Prerequisites
+
+* You have deployed the `nationalparks` back-end application.
+* You have `cluster-admin` or project-level `admin` privileges.
+
+.Procedure
+
+. Create a route to expose the `nationalparks` back-end application by running the following command:
++
+[source,terminal]
+----
+$ oc create route edge nationalparks --service=nationalparks
+----
+
+. Label the `nationalparks` route by running the following command:
```

Comment:
Maybe something like:

As the application requirement, since its _Service_ expects a label `type=parksmap-backend` value to interact with, label the `nationalparks` route by running the following command:

Reply:
I usually like to try to make the actual step pretty concise. I explained a bit more down below the command.

```
The application code expects the `nationalparks` route to be labeled with `type=parksmap-backend`.
```

Not sure if you saw that or not, but we could work on tweaking this wording more if that sounds okay.

Reply:
> I usually like to try to make the actual step pretty concise. I explained a bit more down below the command.
> 
> ```
> The application code expects the `nationalparks` route to be labeled with `type=parksmap-backend`.
> ```
> 
> Not sure if you saw that or not, but we could work on tweaking this wording more if that sounds okay.

Oh, indeed you've commented that down below. So you can ignore this recommendation since your comment explains it well.